pyglmnet/datasets.py

```diff
@@ -47,23 +48,22 @@ def fetch_tikhonov_data(dpath='/tmp/glm-tools'):
     fnames = ['fixations.csv', 'probes.csv', 'spiketimes.csv']
 
     for fname in fnames:
-        url = os.path.join(base_url, "tikhonov/%s" % fname)
+        url = base_url + "/tikhonov/" + fname
```

Comment:
you should use os.path.join so that the correct separator is used for the correct OS

Reply:
Actually, that's a bugfix.  URLs always have '/' but when you're running Windows, os.path.join() substitutes '\'

Reply:
But now I've fixed the flake8 errors

Reply:
Sorry you are right :)